src/rust/ide/view/src/project.rs

```diff
@@ -66,6 +66,12 @@ impl Model {
         }
     }
 
+    /// Quits the IDE.
+    pub fn quit(&self) {
+        info!(self.logger,"Quitting IDE.");
+        std::process::exit(0);
```

Comment:
Wait, how does it work in WASM? What are the echanisms used? Does it work? If so, can you point me to docs about it? I'm wondering how on earth this could exit electron app.